scripts/rff/historical_weather_variability/county/R/heatwave_rasters_to_df.R

```diff
@@ -0,0 +1,65 @@
+rm(list=setdiff(ls(), c("master_path", "code_path")))
+
+dir <- master_path
+setwd(dir)
+
+# HW COUNT
+path <- paste0(dir, "replication/data/heatwaves/4km/annual")
+
+files <- list.files(path)
+files <- files[substr(files, 1,8)=="hw_count"]
+files <- files[nchar(files)==17]
+
+for(file in files){
+  print(file)
+  r <- raster(paste0(path, "/", file))
+  df <- as.data.frame(rasterToPoints(r))
+  if(file==files[1]){data <- df}else{data <- cbind(data, df[,3])}
+}
+
+colnames(data) <-  c("x", "y", paste0("hw_count_", 1981:2020))
+save(data, file =paste0(dir, "replication/data/heatwaves/hw_count_dataframe.Rdata"))
+
+
+# HW LENGTH
+path <- paste0(dir, "replication/data/heatwaves/4km/annual")
+
+files <- list.files(path)
+files <- files[substr(files, 1,9)=="hw_length"]

```

Comment:
Seems a bit fragile/brittle. Helpful comments, e.g. why should nchar(files) be 18?